wg-releases/meeting-notes/2020-05-20.md

```diff
@@ -0,0 +1,47 @@
+# Releases WG
+
+### Date: 2020-05-20
+
+**Members**
+* @codebytere
+* @sofianguy
+* @MarshallOfSound
+* @ckerr
+* @deepak1556
+* @zcbenz
+* @jkleinsc
+* @VerteDinde
+
+**Visitors**
+
+## Agenda
+
+* What should/should not get merged during the quiet week? (@jkleinsc)
+    * What is the goal of thw quiet week?
+    * We should prioritize stability and 
```

Comment:
> We should prioritize stability and

And what? 😆

Reply:
@malept left this for suspense 😅

I don't remember what the rest of the sentence was. So i'll remove the "and"